src/core/views.py

```diff
@@ -3004,6 +3007,11 @@ def view_proposal_review_decision(
 
     if request.POST:
         if 'accept' in request.POST:
+            decision = 'accept'
+        elif 'decline' in request.POST:
+            decision = 'decline'
+
+        if decision == 'accept':
```

Comment:
I don't really understand why this was changed?
`decision` is only assigned the value `'accept'` if `'accept'` is in `request.post`. Doesn't it make sense for this code to just be in the first `if-statement` where this is checked?